src/audio/Audio.js

```diff
@@ -78,13 +78,20 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 		}
 
+		if ( when === undefined || when === 0 ) when = this.context.currentTime;
```

Comment:
As I understand it, passing 0 is equivalent to passing `context.currentTime`. Both mean to play the audio immediately.